### PR TITLE
:zap: add debug log in healthcheck

### PR DESCRIPTION
**Pull request by ahsanv**

### CODE REVIEW

File: packages/cli/config/index.ts [hunk 1]
```diff
@@ -22,6 +22,26 @@ const config = convict({
...
 		},
+		logging: {
+			enabled: {
+				doc: 'Typeorm logging enabled flag.',
+				format: 'Boolean',
+				default: false,
+				env: 'DB_LOGGING_ENABLED',
+			},
+			options: {
+				doc: 'Comma separated logging options, default is "error".',
```
Comment (krynble) [inline packages/cli/config/index.ts:33]: It would be great to have a list of possible values.

File: packages/cli/src/Db.ts [hunk 2]
```diff
@@ -104,10 +104,25 @@ export async function init(): Promise<IDatabaseCollections> {
...
+	let loggingOption: LoggerOptions = (await GenericHelpers.getConfigValue(
+		'database.logging.enabled',
+	)) as boolean;
+
+	if (loggingOption) {
+		const optionsString = (await GenericHelpers.getConfigValue(
+			'database.logging.options',
+		)) as string;
+
+		loggingOption = optionsString.split(',') as LoggerOptions;
```
Comment (krynble) [inline packages/cli/src/Db.ts:116]: I am not sure how it behaves, but perhaps we should `trim()` after splitting just to be on the safe side.

File: packages/cli/config/index.ts [hunk 3]
```diff
@@ -22,6 +22,26 @@ const config = convict({
...
+				format: String,
+				default: 'error',
+				env: 'DB_LOGGING_OPTIONS',
+			},
+			maxQueryExecutionTime: {
+				doc: 'Maximum number of milliseconds query should be executed before logger log a warning. Set 0 to disable',
```
Comment (krynble) [inline packages/cli/config/index.ts:39]: By disabled we mean all queries will be logged or no queries will be logged? I am a bit confused.

**Review by krynble — CHANGES_REQUESTED** [review]
Very useful, I always wanted to enable query logs via environment variable. I always had to change this value manually and rebuild the project 😅 

Added some notes about QoL improvements.

The database logs will be output to console always, right? The `LoggerProxy` instance can log to files too but I don't think this is an option, right?

File: packages/cli/config/index.ts (hunk 3 above)
Comment (ahsanv) [reply packages/cli/config/index.ts:39]: if a query takes longer than specified, there will be warning logged. if set to 0, no logs would be generated. This is separate from enabling log btw, regardless of logs being enabled or not and regardless of the log level, if a query takes longer than specified, warn log is generated.

Comment (ahsanv) [reply packages/cli/config/index.ts:39]: https://typeorm.io/#/logging/enabling-logging

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.161.1`

**Pull request merged**